src/Provider/Mapbox/Mapbox.php

```diff
@@ -347,7 +347,9 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, a
 
             case 'country':
                 $builder->setCountry($value['text']);
-                $builder->setCountryCode(strtoupper($value['short_code']));
+                if (!empty($value['short_code'])) {
```

Comment:
Shouldn't you use `isset()` instead of `!empty()` ?

Reply:
I guess it depends on the code style Geocoder apply because from a result point of view it's the same. 

I'm switching to `isset` + fixing the tests :+1: